glyphs-reader/src/font.rs

```diff
@@ -391,6 +391,43 @@ impl Font {
             });
         }
 
+        // v2 stores values for axes in specific fields, find them and put them into place
+        // "Axis position related properties (e.g. weightValue, widthValue, customValue) have been replaced by the axesValues list which is indexed in parallel with the toplevel axes list."
+        for master in self.font_master.iter_mut() {
+            let mut axis_values = Vec::new();
+            for axis in axes.iter() {
+                let field_name = match axis.tag.as_str() {
+                    "wght" => "weightValue",
+                    "wdth" => "widthValue",
+                    "XXXX" => "customValue",
+                    _ => {
+                        return Err(Error::StructuralError(format!(
+                            "No v2 field known for '{}'",
+                            axis.tag
+                        )))
+                    }
+                };
+                let value = master
+                    .other_stuff
+                    .remove(field_name)
+                    .ok_or_else(|| {
+                        Error::StructuralError(format!(
+                            "Missing '{}' in\n{:#?}",
```

Comment:
you shouldn't expect those axis values are present in the master dict as they were omitted in v2 when equal to their default (100 for weightValue and widthValue, 0 for customValue)
see https://github.com/googlefonts/fontmake-rs/pull/42/files#r1044415236

Reply:
Done